packages/v4/src/content/design-guidelines/components/alert/alert.md

```diff
@@ -4,123 +4,137 @@ section: components
 related: ['Alert', 'Alert group']
 ---
 
-An **alert** is used to notify the user about a change in status or other event without blocking other actions in an interface. These guidelines talk about how to properly use different types of alerts in your designs.
+An **alert** is a notification that provides brief information to the user without blocking their workflow.
+
+There are two main types of alerts:  
+
+- **Inline alerts**: Inline alerts communicate information about a specific user action on a page. An inline alert appears within the content area and disappears when the user closes it or navigates away from the page.
+
+![inline notification placement](./img/inline-notification.png)
+
+- **Toast alerts**: Toast alerts communicate information about an update, confirmation, or other activity. A toast alert overlys content in the upper-right corner of the page and disappears when it times out or when the user dismisses it.
```

Comment:
just noticed a typo here - should be "overlays" not "overly" haha

Reply:
Oh geez